packages/renderer/src/components/molecules/WalletBalance/WalletBalance.tsx

```diff
@@ -25,10 +25,18 @@ import {
 } from "@mantine/core";
 import { ReactComponent as XMRLogo } from "@assets/xmr-logo-1.svg";
 import { ReactComponent as ArrowDown } from "@assets/arrow-down.svg";
+import { useBalances } from "@hooks/haveno/useBalances";
 
 export function WalletBalance() {
   const [isOpen, setOpen] = useState(false);
   const { classes } = useStyles({ isOpen });
+  const { data: availableBalances } = useBalances();
+  const Total = useMemo(() => {
```

Comment:
I'll let this one slide, but don't use PascalCased names for regular variables :)

Reply:
I will follow the advice, thank you very much :)